Cargo.lock

```diff
@@ -12,7 +12,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "kvm-ioctls"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
It's weird that for some crates the `Cargo.lock` file is kept in the subversioning system (tracked in `git`), while for other it isn't.

I know there have been some discussions around which is the best practice. Is there a conclusion? Can we align all crates on this?

Reply:
Yes you're right, I don't know what's the common decision on this and I'll remove it if needed!

Reply:
We agreed on removing them, but I forgot to do so for this crate. @sboeuf do you want to remove it in your PR or should we open a new one?

Reply:
Let me remove the Cargo commit from my PR.